src/api/app/models/staging/staging_project.rb

```diff
@@ -35,7 +35,7 @@ def missing_reviews
       @missing_reviews = []
       attribs = [:by_group, :by_user, :by_project, :by_package]
 
-      (requests_to_review + staged_requests).uniq.each do |request|
+      (requests_to_review | staged_requests.includes(:reviews)).each do |request|
```

Comment:
:cupid:

Reply:
@Ana06 you're just too easy :smile:

Reply:
@dmarcoux I didn't realize that earlier... This only works when combined with fa178200e32a7cfb2897031d19bfb23ba21b176f